### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by osmanwako**
### Inside this PR I did the following task:
- An **owners** table was created with **id** . **age** ,and **full_name** columns.
- A **species** table was created with **id**, and **name** columns.
- An **animals** table was modified to:
    -  An **id**  column is auto increment PRIMARY KEY
    - A Column **species** was removed
    - A column **species_id**  is a foreign key of **species** table
    - A column **owner_id**  is a foreign key of  **owners** table
 - All the above operations were added in the **schema.sql** file.
 - Insert the following data into the **owners** table:
    - The given data were inserted into **owners** table.
- Insert the following data into the species table:
    - **Pokemon** and **Digimon** were added into **species** table.
- Modify your inserted animals so it includes the species_id value:
      - If the name ends in "mon" it will be Digimon
m1
    - All other animals are Pokemon
m2
- Modify your inserted animals to include owner information (owner_id):
     - Sam Smith owns Agumon.
m3
     - Jennifer Orwell owns Gabumon and Pikachu.
m4
     - Bob owns Devimon and Plantmon.
m5
     - Melody Pond owns Charmander, Squirtle, and Blossom.
m6
     - Dean Winchester owns Angemon and Boarmon.
m7
- All insertions and modifications that happened were added in the **data.sql** file.
- **Write queries (using JOIN) to answer the following questions:**
     - What animals belong to Melody Pond?
j1
     - List of all animals that are Pokemon (their type is Pokemon).
j2
     - List all owners and their animals, remember to include those that don't own any animal.
j3
     - How many animals are there per species?
j4
     - List all Digimon owned by Jennifer Orwell.
j5
     - List all animals owned by Dean Winchester that haven't tried to escape.
j6
     - Who owns the most animals?
j7
- All the above join queries were added in the  **queries.sql** file.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -331,4 +331,107 @@ FROM
...
+  a.id as animal_id,
+  a.name as animal_name,
+  a.escape_attempts,
+  o.full_name as owner_name
+FROM
+  animals a
+  JOIN owners o ON a.owner_id = o.id
+WHERE
+  a.escape_attempts = 0
+  AND o.full_name = 'Sam Smith';
```
Comment (Ghiftee) [inline queries.sql]: - [ ] Here, you are required to list all animals owned by `Dean Winchester` that have not tried to escape and not `Sam Smith`. Your screenshot has the correct query and results, however, kindly correct it here and remove the screenshot for `Sam Smith`

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @osmanwako 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Commands are placed in the right SQL files
✔️ Screenshots are added
✔️ Queries are correct

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review 👇🏽 _

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 1 above)
Comment (osmanwako) [reply queries.sql]: Hi @Ghiftee 
yah! Really, Impressive Review. I will remove it.
You noticed me well. 
thank you very much

**Review by Olamarx — APPROVED** [review]
Hi @osmanwako, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Required change fixed ✔️
- All queries, data, and schema properly set ✔️
- All data properly queried ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**